Covid Lunches/express/express-bootstrap/public/testing/sample_locations.js

```diff
@@ -0,0 +1,7 @@
+// JavaScript source code
+var testLocations = [["Sample school 1", 39.015975363512396, -76.67026638671018],
```

Comment:
Will add sample data to database BUT we should keep these sample locations as well in order to keep our tests separate and independent.